rpc_jobs/standard_job_fi.yml

```diff
@@ -0,0 +1,139 @@
+- job-template:
+    name: 'FI_{repo_name}-{branch}'
```

Comment:
I can't remember if this was discussed, but do we want to allow components to spawn multiple different FI/L3 tests?  I've stripped out`{image}`, `{scenario}`, and `{action}` but can re-add if there is scope for projects to run multiple of these long-running tests prior to release.